src/Time/Bool.hs

```diff
@@ -0,0 +1,39 @@
+{-# LANGUAGE AllowAmbiguousTypes        #-}
+{-# LANGUAGE ConstraintKinds            #-}
+{-# LANGUAGE CPP                        #-}
+{-# LANGUAGE DataKinds                  #-}
+{-# LANGUAGE DeriveGeneric              #-}
+{-# LANGUAGE ExplicitForAll             #-}
+{-# LANGUAGE FlexibleContexts           #-}
+{-# LANGUAGE InstanceSigs               #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE ScopedTypeVariables        #-}
+{-# LANGUAGE TypeApplications           #-}
+{-# LANGUAGE TypeFamilies               #-}
+{-# LANGUAGE TypeInType                 #-}
+{-# LANGUAGE TypeOperators              #-}
+{-# LANGUAGE UndecidableInstances       #-}
```

Comment:
Okay, now I understand that 15 pragmas per module is too much even for me... Could you please move extensions we have in every module into `default-extensions` inside `.cabal` file? Also I think that `CPP` `TypeInType` `AllowAmbiguosTypes` (and probably something else) is not needed in that module... Unfortunately, GHC cannot tell which extensions are unused so this requires some manual extra work :(